source/ui/BioXAS/BioXASPersistentView.cpp

```diff
@@ -38,6 +38,13 @@ BioXASPersistentView::BioXASPersistentView(QWidget *parent) :
 
 	setLayout(layout);
 
+	// Create SR1 current view.
+
+	BioXASControlEditor *sr1CurrentEditor = new BioXASControlEditor(CLSStorageRing::storageRing()->ringCurrentControl());
+	sr1CurrentEditor->setTitle("SR1 Current");
```

Comment:
I can't seem to find where read-only is set to true.  Is there another mechanism that can set it to true?